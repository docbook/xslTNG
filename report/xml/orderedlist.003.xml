<?xml version="1.0" encoding="utf-8"?>
<article xmlns="http://docbook.org/ns/docbook" version="5.0">
<title>Unit test: orderedlist.003.html</title>

<para>Test list continuation. And cross-references;
item “2”: <xref linkend="i2"/>,
item “4”: <xref linkend="i4"/>, and
item “5.3”: <xref linkend="i5.3"/>.</para>

<orderedlist spacing="compact">
   <listitem>
      <simpara>First item</simpara>
   </listitem>
   <listitem xml:id="i2">
      <simpara>Second item</simpara>
   </listitem>
   <listitem>
      <simpara>Third item</simpara>
   </listitem>
</orderedlist>

<para>And then a break occurs...a break in my sanity, apparently,
because changing the numeration style across a “continues” break seems
really stupid. But this a unit test, after all. Stupid is
allowed.</para>

<orderedlist numeration="upperalpha" continuation="continues" spacing="compact">
   <listitem xml:id="i4">
      <simpara>Fourth item</simpara>
   </listitem>
   <listitem>
      <simpara>Fifth item</simpara>
      <orderedlist spacing="compact">
        <listitem>
          <simpara>Fifth/First item</simpara>
        </listitem>
        <listitem>
          <simpara>Fifth/Second item</simpara>
        </listitem>
        <listitem xml:id="i5.3">
          <simpara>Fifth/Third item</simpara>
        </listitem>
      </orderedlist>
   </listitem>
   <listitem>
      <simpara>Sixth item</simpara>
   </listitem>
</orderedlist>

</article>
