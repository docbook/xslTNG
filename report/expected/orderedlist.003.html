<!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" class="no-js"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><title>Unit test: orderedlist.003.html</title><meta name="viewport" content="width=device-width, initial-scale=1.0"/><link href="https://purl.org/dc/elements/1.1/" rel="schema.dc"/><meta content="2011-04-22T17:02:00-06:00" name="dc.modified"/><meta content="DocBook xslTNG" name="generator"/><link href="./css/docbook.css" rel="stylesheet" media="screen"/></head><body class="home"><nav class="top"></nav><main><article class="article component"><header><h1>Unit test: orderedlist.003.html</h1></header><p>Test list continuation. And cross-references;
item “2”: <a href="#i2" class="xref xref-listitem"><span class="label">2</span></a>,
item “4”: <a href="#i4" class="xref xref-listitem"><span class="label">D</span></a>, and
item “5.3”: <a href="#i5.3" class="xref xref-listitem"><span class="label">E<span class="sep">.</span>c</span></a>.</p><ol class="orderedlist" type="1"><li><p class="simpara">First item</p></li><li id="i2"><p class="simpara">Second item</p></li><li><p class="simpara">Third item</p></li></ol><p>And then a break occurs...a break in my sanity, apparently,
because changing the numeration style across a “continues” break seems
really stupid. But this a unit test, after all. Stupid is
allowed.</p><ol class="orderedlist" start="4" type="A"><li id="i4"><p class="simpara">Fourth item</p></li><li><p class="simpara">Fifth item</p><ol class="orderedlist" type="a"><li><p class="simpara">Fifth/First item</p></li><li><p class="simpara">Fifth/Second item</p></li><li id="i5.3"><p class="simpara">Fifth/Third item</p></li></ol></li><li><p class="simpara">Sixth item</p></li></ol></article></main><nav class="bottom"></nav></body></html>